docs/guide/writing/style-guides.md

```diff
@@ -31,7 +30,7 @@ Classic style guides for software documentation include:
 Other useful resources:
 
 - [18F US Government Content Guide](https://content-guide.18f.gov)
-- [DigitalOcean Documentation Style Guide](https://docs.digitalocean.com/style/
+- [DigitalOcean Documentation Style Guide](<https://docs.digitalocean.com/style/>
```

Comment:
I think you want 

[DigitalOcean Documentation Style Guide](https://docs.digitalocean.com/style/)